dataverse/orgsvc/C#-NETCore/ElasticTables/ElasticTableSamples/ElasticTableSamples/Utility.cs

```diff
@@ -0,0 +1,248 @@
+
+            Console.WriteLine($"Creating {DeviceIdSchemaName} attribute..\n");
+            CreateAttributeRequest createDeviceIdAttributeRequest = new CreateAttributeRequest
+            {
+                EntityName = SensorDataSchemaName.ToLower(),
+                Attribute = new StringAttributeMetadata
+                {
+                    SchemaName = DeviceIdSchemaName,
+                    DisplayName = new Label("Device Id", 1033),
+                    MaxLength = 1000,
+                }
+            };
+            client.Execute(createDeviceIdAttributeRequest);
+            Console.WriteLine($"Created {DeviceIdSchemaName} attribute.\n");
+
+            Console.WriteLine($"Creating {ValueSchemaName} attribute..\n");
+            CreateAttributeRequest createValueAttributeRequest = new CreateAttributeRequest
+            {
+                EntityName = SensorDataSchemaName.ToLower(),
+                Attribute = new IntegerAttributeMetadata
+                {
+                    SchemaName = ValueSchemaName,
+                    DisplayName = new Label("Value", 1033),
+                }
+            };
+            client.Execute(createValueAttributeRequest);
+            Console.WriteLine($"Created {ValueSchemaName} attribute.\n");
+
+            Console.WriteLine($"Creating {TimeStampSchemaName} attribute..\n");
+            CreateAttributeRequest createTimeStampAttributeRequest = new CreateAttributeRequest
+            {
+                EntityName = SensorDataSchemaName.ToLower(),
+                Attribute = new DateTimeAttributeMetadata()
+                {
+                    SchemaName = TimeStampSchemaName,
+                    DisplayName = new Label("Time Stamp", 1033),
+                }
+            };
+            client.Execute(createTimeStampAttributeRequest);
+            Console.WriteLine($"Created {TimeStampSchemaName} attribute.\n");
+
+            Console.WriteLine($"Creating {EnergyConsumptionSchemaName} attribute..\n");
+            CreateAttributeRequest createSensorDataAttributeRequest = new CreateAttributeRequest
+            {
+                EntityName = SensorDataSchemaName.ToLower(),
+                Attribute = new StringAttributeMetadata
+                {
+                    SchemaName = EnergyConsumptionSchemaName,
+                    RequiredLevel = new AttributeRequiredLevelManagedProperty(AttributeRequiredLevel.None),
+                    MaxLength = 1000,
+                    FormatName = StringFormatName.Json,
+                    DisplayName = new Label("Energy Consumption", 1033),
+                    Description = new Label("Stores unstructured energy consumption data as reported by device", 1033)
+                }
+            };
+
+            client.Execute(createSensorDataAttributeRequest);
+            Console.WriteLine($"Created {EnergyConsumptionSchemaName} attribute.\n");
+        }
+
+        internal static void DeleteSensorDataEntity(ServiceClient client)
+        {
```

Comment:
RE: 
> `DeleteSensorDataEntity(ServiceClient client)`

s/b `DeleteSensorDataEntity(IOrganizationService service)`

For the `CreateSensorDataEntity` method I understand using `ServiceClient` b/c you need to use `ExecuteWebRequest`. 

But we should use `IOrganizaitonService` any other time.